include/eve/module/complex/regular/detail/math.hpp

```diff
@@ -10,18 +10,247 @@
 #include <eve/module/core.hpp>
 #include <eve/module/math.hpp>
 #include <eve/module/complex/regular/traits.hpp>
+#include <eve/module/complex/regular/conj.hpp>
+#include <eve/module/complex/regular/i.hpp>
+#include <eve/module/complex/regular/exp_i.hpp>
+#include <eve/module/complex/regular/exp_ipi.hpp>
 
 namespace eve::detail
 {
-  EVE_FORCEINLINE auto complex_binary_dispatch( eve::tag::ulpdist_
-                                              , auto const& z1, auto const& z2
+
+  //==============================================================================================
+  // sqrt cosh cos acosh asinh atan exp exp_i exp_ipi log rec
```

Comment:
sqrt, exp_i and exp_ipi are right there ...

Reply:
forgot the point at the end of the line